src/service/api/article.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+
+const {Router} = require(`express`);
+const {HTTP_STATUS_CODE} = require(`../../constants`);
+const {articleValidator} = require(`../middlewares`);
+
+const articleRouter = new Router();
+
+const initArticlesApi = (app, service) => {
```

Comment:
```suggestion

const initArticlesApi = (app, service) => {
const articleRouter = new Router();

```
перемести инициализацию в функцию (нужно исправить для всех роутов)